src/register/register.ts

```diff
@@ -30,9 +32,20 @@ export class Register {
    */
   private static registers: { [key: string]: IRegisterContent } = {
     '"': { text: "", registerMode: RegisterMode.CharacterWise },
-    '*': { text: "", registerMode: RegisterMode.CharacterWise }
+    '*': { text: "", registerMode: RegisterMode.CharacterWise, isClipboardRegister: true },
+    '+': { text: "", registerMode: RegisterMode.CharacterWise, isClipboardRegister: true }
   };
 
+  public static isClipboardRegister(registerName: string): boolean {
+    const register = Register.registers[registerName];
+
+    if (register && register.isClipboardRegister) {
```

Comment:
I actually tried that initially, but tsc doesn't like it very much:

```
> vim@0.1.10 vscode:prepublish c:\Users\Ben\Code\vscode-vim
> node ./node_modules/vscode/bin/compile -p ./
src/register/register.ts(42,12): error TS2322: Type 'boolean | undefined' is not assignable to type 'boolean'.
  Type 'undefined' is not assignable to type 'boolean'.
```

It works fine if I change the signature to `isClipboardRegister(string): boolean | undefined`, but I wasn't sure if that was preferable to having a messier function body.